src/modules/sdl/consumer_sdl_audio.c

```diff
@@ -356,19 +355,33 @@ static int consumer_play_audio( consumer_sdl self, mlt_frame frame, int init_aud
 	if ( init_audio == 0 )
 	{
 		mlt_properties properties = MLT_FRAME_PROPERTIES( frame );
-		bytes = ( samples * channels * 2 );
+		int bytes_copied = 0;
+		int bytes_to_copy = ( samples * channels * sizeof( int16_t ) );
```

Comment:
sizeof( *pcm ) is preferable over a named type, but that is a nit. Only change perhaps if making other changes.